### DISCUSSION TITLE
Headstone state questions

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [muninn/graves#49] [Q&A]
I've been thinking about my use cases for headstone states. Here's how I'm thinking of using various states (_and some questions_).  Feedback welcome.

1. A headstone was planned but has not been built, or may never be built - '**[Unbuilt](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Unbuilt)**'
2. A headstone has been built but is not on display - '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**' 
3. A headstone has been built and functioning as planned - '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**'
4. A headstone is functioning as planned but a tree is threatening to topple it over - '**At Risk**' #48 
6. A headstone was removed from a cemetery and placed in storage - '**[Abandoned](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Abandoned)**' (_or is this still '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**'?_)
7. A headstone was replaced by another. The original headstone is on display in a Museum - '**[Abandoned](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Abandoned)**' (_or should this be '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**' as the original headstone is still performing its function i.e. being a physical feature commemorating a person or group of people who have died._)
8. A headstone was removed from a cemetery and placed in storage, then later destroyed and buried elsewhere - '**[Demolished](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Demolished)**'
9. Parts of demolished headstone were discovered and linked to human remains. The full inscription is not visible - '**[Demolished](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Demolished)**', with a **[hasLocation](https://rdf.muninn-project.org/ontologies/graves-en.html#term_hasLocation)** property to indicate it has been found 
10. Parts of demolished headstone were discovered and linked to human remains. The full inscription is visible - '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**', with a **hasLocation** property to indicate it has been found. (_Is the fact that the inscription is fully visible the thing that causes the status to change, as the headstone function is now being fully performed?_)  
11. Parts of demolished headstone were discovered, but cannot be linked to human remains. It may have been put on display - '**[Ruins](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Ruins)**' 
12. A demolished headstone was rebuilt and put on display - '**[Rebuilt](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Rebuilt)**' with dates (_as a transitory status to '**[Serviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Servicable)**'?_)
13. A headstone was functioning as planned but weathering has removed the inscription. The human remains marked by the headstone are known but no record of the inscription exists - '**[Unserviceable](https://rdf.muninn-project.org/ontologies/graves-en.html#term_UnServicable)**' 
14. A headstone was functioning as planned but weathering has removed the inscription. The human remains marked by the headstone are not known and no record of the inscription exists and hence it cannot be repaired - '**[Ruins](https://rdf.muninn-project.org/ontologies/graves-en.html#term_Ruins)**'

**Comment by Stephen-Gates** [comment]
Acknowledging the conversation in https://github.com/muninn/graves/issues/48 about `hasState` values being originally intended for Machinery, which is why there is a value of `Burned`, but in my mind that is the Cause of an Object being `Unserviceable`. 

`hasState` applies to `Monument` (Headstone, etc.), `Dead_people_place` (Cemetery, etc.) and `Container_for_human_remains` (Grave, etc.). These could all have a different set of causes why they become `Unserviceable`. 

The Cause of a `Monument` being `Unserviceable` could be: `Collapsed`, `Vandalised`, `Weathered`, `Being Repaired`. These values may not apply to all sub-classes of `Dead_people_place`. E.g. A `Columbarium` may collapse but a `Cemetery` probably won't. 

While the set of `hasState` values, with the addition of `Threatened`, will support my needs, I'm wondering if the Cause of `hasState`, needs to be a separate property and if it needs to vary by class or sub-class? 

You could go further a split out `Threatened` from `hasState`. For example: `hasState` = `Serviceable`; `threatenedBy` = `Vegetation`, `Land Subsidence`, etc.

**Reply by rwarren2** [reply]
The whole 'state' thing is something that I've never been completely happy with.... 

How would you feel about using the provenance ontology for something along the lines of:
OvergrowingVegetations ( Activity) ->  prov:influences -> Monument
